apps/modernization-ui/src/apps/patient/profile/Demographics.tsx

```diff
@@ -1,16 +1,16 @@
 import { Grid } from '@trussworks/react-uswds';
-import { NamesTable } from 'pages/patient/profile/names';
-import { AddressesTable } from 'pages/patient/profile/addresses';
-import { PhoneAndEmailTable } from 'pages/patient/profile/phoneEmail';
-import { IdentificationsTable } from 'pages/patient/profile/identification';
```

Comment:
This should be under the components folder under the patient
That is how the page builder is. 

Under the apps folder's app we should have the component, models and all other folders. There is where we need to add these files.